packages/clarity-decode/src/core.ts

```diff
@@ -0,0 +1,10 @@
+import { Data, Layout } from "clarity-js";
+
+export function expand(input: string): [string, string] {
```

Comment:
Maybe demangle would be clearer? Exapnd is very vague and generic.

Reply:
Agree, renamed.